### PR TITLE
Add InvoxiPlayGames's tool

### PR DESCRIPTION
**Pull request by JMK19**
This makes the guide simpler and easier to do for normal users. Original code is saved as legacy.html in /guides/jbmestar if you would like to reference it. Feel free to delete it.

### CODE REVIEW

File: jbmestar/index.html [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    <script src="../../src/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+
+      integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+
+      crossorigin="anonymous">
+    <script src="../../src/js/scrollToTop.js"></script>
+    <link href="../../src/stylesheets/bootstrap.min.css" rel="stylesheet" type="text/css">
+    <link href="../../src/stylesheets/style.css" rel="stylesheet" type="text/css">
+    <meta name="site-name" content="Jailbreak Guide - Example Guide">
```
Comment (The-Hacker894) [inline jbmestar/index.html]:
```suggestion
    <meta name="site-name" content="Jailbreak Guide - Star">
```

Replace the example placeholder stuff

File: jbmestar/index.html [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    <meta name="description" content="Jailbreak Guide Example Template">
+    <meta name="keywords" content="Help on Jailbreak, I need Jailbreak help, help Jailbreak, help, Jailbreaking Help, Jailbreak Guides, Jailbreak, Guides, How to Jailbreak, Jailbreak iPhone, Jailbreak iPad, How to Jailbreak iPhone, How to Jailbreak iPad, how to Jailbreak iOS, How to Jailbreak Apple">
```
Comment (The-Hacker894) [inline jbmestar/index.html]: These meta tags needed updating

File: jbmestar/index.html [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    <meta name="author" content="anon">
```
Comment (The-Hacker894) [inline jbmestar/index.html]: Don't want to add yourself as the author?

File: jbmestar/index.html (hunk 1 above)
Comment (The-Hacker894) [reply jbmestar/index.html]: Keep in mind this isn't the only occurrence that needs to be changed.

**Comment by JMK19** [comment]
Jeez, a lot of changes must be made before this. Thanks for looking out! Working on it RN!

**Comment by JMK19** [comment]
I've fixed these issues with a new push.

**Comment by The-Hacker894** [comment]
Everything looks alright now. Will be merging.

**Pull request merged**